src/nyc_trees/apps/survey/fixtures/species.json

```diff
@@ -108,12 +108,12 @@
         "model": "survey.Species",
         "fields": {
             "species_code": "ACPLCR",
-            "scientific_name": "Acer platanoides",
+            "scientific_name": "Acer platanoides 'Crimson King'",
             "created_at": "2023-04-10T17:11:50.350Z",
             "updated_at": "2023-04-10T17:11:50.350Z",
             "forms_id": 92393,
-            "common_name": "Crimson King Norway Maple",
-            "cultivar": "Not Green Leaf"
+            "common_name": "Crimson King Maple",
+            "cultivar": ""
```

Comment:
I did that so that the cultivar text wouldn't appear in the species dropdown, in order to match the 'treecorder list' format. Should we just modify the rendering layer to never show cultivars? The ones changed here are the only ones with cultivar values.